src/spectrecoff/Spectre/Progress.fs

```diff
@@ -45,16 +45,13 @@ let defaultTemplate =
     |> withProgressBarColumn
     |> withPercentageColumn
 
-let startCustom template (operation: ProgressOperation) =
+let startCustom template (operation: ProgressOperation<'T>) =
     task {
-    return! AnsiConsole
-        .Progress()
-        .Columns(template.Columns |> List.toArray)
-        .StartAsync(operation)
-}
-
-let start =
```

Comment:
Why did you remove the function with the baked in default? I'd vote to keep it.

Reply:
Re-added it, but it seems we need to pass `()` to make it a function:

![image](https://github.com/user-attachments/assets/20a1edfe-7ce8-495a-8886-cf7e53257134)

Reply:
In fact, I pushed the change on this branch ;)

Reply:
I see, thanks :)